src/ump/api/job.py

```diff
@@ -79,7 +81,8 @@ def create(
             process_title,
             name,
             parameters,
-            user_id = user
+            user_id = user,
+            ensemble_id = ensemble_id
```

Comment:
Well, as implemented now, jobs are only created once an ensemble is executed. Maybe it would make sense to separate the job configuration from the job execution data, that way the job configurations could be assigned to multiple ensembles and the job executions could be assigned to their job. What do you think?

Reply:
Yes I think that would make sense. Then users can also create "empty" ensembles and manually add jobs which results were already calculated to the ensembles, but can also configure bunch of jobs that will be executed for an ensemble.

Reply:
https://github.com/citysciencelab/urban-model-platform/issues/39